packages/farcaster-js-neynar/src/v2/NeynarV2APIClient.ts

```diff
@@ -315,12 +321,12 @@ export class NeynarV2APIClient {
    */
   public async reactToCast(
     signerUuid: string,
-    reaction: ReactionType,
+    reaction: 'like' | 'recast',
```

Comment:
Noting that this would necessitate a major version bump.

Is this just to reduce the friction from developers having to import ReactionType from the `/v2` path?

Reply:
Yeah that's right